.gitmodules

```diff
@@ -83,3 +83,11 @@ ignore = dirty
 depth = 1
 shallow = true
 fetchRecurseSubmodules = false
+[submodule "test/node/node"]
+	path = test/node/node
+	url = https://github.com/nodejs/node.git
```

Comment:
Can we lock this to a release hash instead of main.

Reply:
It's pinned to a specific commit

Reply:
yeah git submodules are always a specific commit